proxy/src/http/conn_pool.rs

```diff
@@ -294,6 +294,7 @@ async fn connect_to_compute_once(
         .password(&conn_info.password)
         .dbname(&conn_info.dbname)
         .max_backend_message_size(MAX_RESPONSE_SIZE)
+        .connect_timeout(time::Duration::from_secs(2))
```

Comment:
Do you mind wrapping `2` into constant as you did w/ `NUM_RETRIES_WAKE_COMPUTE`?